README.md

```diff
@@ -112,12 +112,119 @@ devtools::install_github("UBC-MDS/mealprepR")
 
 ## Examples
 
+To load the package:
+
+``` r
+library(mealprepR)
+library(devtools)
+load_all()
+```
+
```

Comment:
@SamEdwardes , when I updated my local version of the README with your latest updates (including your code chunks for the `make_recipe` example), I wasn't able to knit the file because running `library(mealprepR)` alone (without the `library(devtools)` and `load_all()`) would not import the functions. It kept saying that it couldn't find the functions, so I had to add this code chunk to make the following code chunks work. Were you able to use your function just by running `library(mealprepR)`?